README.md

```diff
@@ -1,10 +1,64 @@
-vcf<img src="http://i.giphy.com/R6X7GehJWQYms.gif" width="30">maf
-=======
+# vcf2maf
```

Comment:
I modified this to vcf2maf because else Docker Hub has trouble displaying this title.